README.md

```diff
@@ -1 +1,14 @@
 # Message Queue Consumer
```

Comment:
This REAME should follow the style guide at https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md